### ISSUE TITLE
Support .titlebar class for headerbars

### ISSUE THREAD

**Issue by cassidyjames** [system76/pop-gtk-theme#43]
It's a convention in elementary and AppCenter apps to use a `.titlebar` class on the headerbar to be able to reduce the padding and make it look like a normal titlebar while still packing in some buttons. elementary Terminal, for example, uses this for its search and font-size buttons.

Something like this would support that convention:

```
headerbar.titlebar {
	padding-top: 0;
	padding-bottom: 0;
}

headerbar.titlebar button {
	margin-top: 0;
	margin-bottom: 0; 
}
```

That takes elementary Terminal in Pop!_OS from this:

![screenshot from 2017-10-04 22-14-54](https://user-images.githubusercontent.com/611168/31210721-878f9db2-a951-11e7-81f2-11172a0dd7cb.png)

...to this:

![screenshot from 2017-10-04 22-15-10](https://user-images.githubusercontent.com/611168/31210734-98a399a0-a951-11e7-8bd3-db22593ece90.png)

...which is more how it's designed to look.

**Comment by cassidyjames** [comment]
Re-opening this since the previous fix broke other things and has been reverted.

**Comment by isantop** [comment]
Re-fixed in 9498c5f4b0a7f6c3231ec59ee529982f5cdb93cb
